src/js/controller/controller.js

```diff
@@ -408,7 +408,14 @@ Object.assign(Controller.prototype, {
             }
 
             const playReason = _getReason(meta);
-            const startTime = meta ? meta.startTime : null;
+            let startTime;
+            
+            if (meta && meta === Object(meta)) {
+                startTime = meta.startTime = isValidNumber(meta.startTime) ? meta.startTime : _model.get('playlistItem').starttime;
+            } else {
```

Comment:
This probably is not the right place to read `playlistItem.starttime`.  Why don't we always add `playlistItem.starttime` to the `MEDIA_BEFOREPLAY` when it exists on the playlistItem (unless overwritten by meta)?